apps/multiplatform/common/src/commonMain/kotlin/chat/simplex/common/model/SimpleXAPI.kt

```diff
@@ -1554,7 +1554,7 @@ object ChatController {
 
   suspend fun receiveFiles(rhId: Long?, user: UserLike, fileIds: List<Long>, userApprovedRelays: Boolean = false, auto: Boolean = false) {
     val filesAccepted = mutableListOf<AChatItem>()
-    val filesIdsToApprove = mutableStateListOf<Long>()
+    val fileIdsToApprove = mutableStateListOf<Long>()
```

Comment:
why is it `mutableStateListOf` instead of `mutableListOf`? There is no `@Composable` function

Reply:
didn't notice, right